sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,319 @@
+| [`SHOW MASTER STATUS`](/sql-statements/sql-statement-show-master-status.md) | Shows the latest TSO in the cluster. |
+| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
+| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows the current sessions connected to the same TiDB server. |
+| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Shows query profiles. Included for [compatibility with MySQL](/mysql-compatibility.md). Currently only returns an empty result. |
+
+</CustomContent>
+
+## Locking statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`LOCK STATS`](/sql-statements/sql-statement-lock-stats.md) | Locks statistics of tables or partitions. |
+| [`LOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | Locks tables for the current session. |
+| [`UNLOCK STATS`](/sql-statements/sql-statement-unlock-stats.md) | Unlocks statistics of tables or partitions. |
+| [`UNLOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | Unlocks tables. |
+
+## Account management / Data Control Language
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) | Modifies a user. |
+| [`CREATE ROLE`](/sql-statements/sql-statement-create-role.md) | Creates a role. |
+| [`CREATE USER`](/sql-statements/sql-statement-create-user.md) | Creates a new user. |
+| [`DROP ROLE`](/sql-statements/sql-statement-drop-role.md) | Drops an existing role. |
+| [`DROP USER`](/sql-statements/sql-statement-drop-user.md) | Drops an existing user. |
+| [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) | Reloads the in-memory copy of privileges from the privilege tables. |
+| [`GRANT <privileges>`](/sql-statements/sql-statement-grant-privileges.md) | Grants privileges. |
+| [`GRANT <role>`](/sql-statements/sql-statement-grant-role.md) | Grants a role. |
+| [`RENAME USER`](/sql-statements/sql-statement-rename-user.md) | Renames an existing user. |
+| [`REVOKE <privileges>`](/sql-statements/sql-statement-revoke-privileges.md) | Revokes privileges. |
+| [`REVOKE <role>`](/sql-statements/sql-statement-revoke-role.md) | Revokes a role. |
+| [`SET DEFAULT ROLE`](/sql-statements/sql-statement-set-default-role.md) | Sets a default role. |
+| [`SET PASSWORD`](/sql-statements/sql-statement-set-password.md) | Changes a password. |
+| [`SET ROLE`](/sql-statements/sql-statement-set-role.md) | Enables roles in the current session. |
+| [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md) | Shows the `CREATE` statement for a user. |
+| [`SHOW GRANTS`](/sql-statements/sql-statement-show-grants.md) | Shows privileges associated with a user. |
+| [`SHOW PRIVILEGES`](/sql-statements/sql-statement-show-privileges.md) | Shows available privileges. |
+
+<CustomContent platform="tidb">
```

Comment:
The `CustomContent` tag currently does support wrap section titles.  If we use it to wrap sections, the content looks fine in the docs website, but the section title "## TiDB Binlog & TiCDC" will still be displayed in the "What's on this page" pane for TiDB Cloud 😞.   Perhaps, we need to keep this section for both TiDB and TiDB Cloud and add a note after the section title stating the following statements are not applicable to TiDB Cloud.

Reply:
This section list multiple statements that are not in the Cloud TOC...

Maybe we should rename the section to "Replication statements" and then point to https://docs.pingcap.com/tidbcloud/changefeed-overview and say that TiDB Cloud uses a web interface and API instead of statements?

Reply:
Or we could simply leave it and only list `SHOW MASTER STATUS`? Maybe that's best.

Reply:
Done in 1597d9d37b8ae9af0bdc553c697942cb9a069f90